docs/user-guide/cli-installcli.md

```diff
@@ -61,27 +61,31 @@ If you have restricted internet access at your site, use the following method to
 4. Issue the following command against the extracted directory to install Zowe CLI on your computer:
 
     ```
-    npm install -g zowe-cli.tgz 
+    npm install -g zowe-cli.tgz
     ```
```

Comment:
This tip works well elsewhere, but if I remove the ticks from this instance does _not_ become a code snippet. I think that this is because it is indented underneath an ordered list.